### PR TITLE
Add Carousel functionality  to data-cards when in mobile view

### PR DESCRIPTION
**Pull request by eds-kunal**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #331

Test URLs:
- Before: https://main--world-bank--aemsites.aem.live/ext/en/home
- After: https://data-cards-carousel--world-bank--aemsites.aem.live/ext/en/home

Here added Carousel Functionality to data-cards when in mobile view.

### CODE REVIEW

File: blocks/data-cards/data-cards.js [hunk 1]
```diff
@@ -25,6 +27,92 @@ function createRTEContainer(rteElements) {
...
+  const carouselItems = document.querySelector('.data-card-items');
+  if (option === '+') {
+    carouselItems.scrollLeft += itemWidth;
+  } else {
+    carouselItems.scrollLeft -= itemWidth;
+  }
+}
+
+function buttonEvents(nextBtn, prevBtn) {
+  const cardsContainer = document.querySelector('.data-card-items');
```
Comment (thomsebastin) [inline blocks/data-cards/data-cards.js:62]: avoid referencing `document` wherever possible and target `block`.

Comment (eds-kunal) [reply blocks/data-cards/data-cards.js]: Updated @thomsebastin.

File: blocks/data-cards/data-cards.js [hunk 2]
```diff
@@ -25,6 +27,92 @@ function createRTEContainer(rteElements) {
...
+
+function buttonEvents(nextBtn, prevBtn, block) {
+  const cardsContainer = block.querySelector('.data-card-items');
+  const moveRightBtn = block.querySelector(`.button-${nextBtn}`);
+  const moveLeftBtn = block.querySelector(`.button-${prevBtn}`);
+  const screenWidth = window.screen.width;
+  let currentIndex = 0;
+  let maxIndex = cardsContainer.children.length;
+
+  if (screenWidth > 600) {
```
Comment (mkbansal1) [inline blocks/data-cards/data-cards.js:69]: why we are considering 600 or 700 px ? mobile viewport we are supporting is below 768px

**Review by mkbansal1 — COMMENTED** [review]
Slide behaviour is removed, ideally both sliding and allows should work
Last slide border right is missing
image

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
Please check the comments

File: blocks/data-cards/data-cards.js (hunk 1 above)
Comment (mkbansal1) [reply blocks/data-cards/data-cards.js]: I still see document.query at multiple place Line#53, Line#62,#63, #64

**Review by thomsebastin — APPROVED** [review]

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**